src/utils/ubus.c

```diff
@@ -1148,11 +1153,20 @@ void update_tcp_connections(struct uloop_timeout *t) {
     uloop_timeout_set(&tcp_con_timer, timeout_config.update_tcp_con * 1000);
 }
 
+void check_tcp_timeouts(struct uloop_timeout *t) {    
+    dawnlog_debug_func("Entering...");
+
+    check_tcp_timeout(TCP_TIMEOUT);
+
+    uloop_timeout_set(&tcp_timeout_timer, TCP_TIMEOUT * 1000);
```

Comment:
timer trigger interval `TCP_TIMEOUT` ?

Reply:
Why not?

Reply:
TCP_TIMEOUT means the timeout for every tcp connection

timer interval is the rate to trigger timter, should use a better interval time, maybe every 5s or what.

Reply:
I made the timer configurable.

Reply:
I made now the timeout configurable. Can you test it and than we can merge?